crates/ethereum/payload/src/lib.rs

```diff
@@ -239,10 +240,43 @@ where
             requests_root,
         };
 
+        // Create an EVM instance
+        let mut evm = self.evm_config.evm(db);
```

Comment:
this is an empty block, so we simply need to return no receipts here, no need to execute anything